sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.institutions.controller.ts

```diff
@@ -63,4 +68,92 @@ export class ApplicationOfferingChangeRequestInstitutionsController extends Base
       count: applications.count,
     };
   }
+
+  /**
+   * Gets all in progress application offering request changes.
+   * @param locationId location id.
+   * @param paginationOptions options to execute the pagination.
+   * @returns list of inprogress application offering request changes.
+   */
+  @HasLocationAccess("locationId")
+  @Get("in-progress")
+  async getInprogressApplications(
+    @Param("locationId", ParseIntPipe) locationId: number,
+    @Query() pagination: OfferingChangePaginationRequestedOptionsAPIInDTO,
```

Comment:
Please change the `@param` comment to pagination or change this variable to paginationOptions

Reply:
Likewise can be done for the `getEligibleApplications` method as well.